app/views/dashboards/followers.html.erb

```diff
@@ -3,10 +3,10 @@
 <div id="index-container"
   data-params="<%= params.to_json(only: %i[which per_page]) %>"
   data-which="followers">
-    <header class="crayons-layout">
-      <h1 class="p-2 pb-0 m:p-0 crayons-title">Dashboard &raquo; Followers</h1>
+    <section class="crayons-layout" aria-labelledby="header-title">
+      <h1 class="p-2 pb-0 m:p-0 crayons-title" id="header-title">Dashboard &raquo; Followers</h1>
       <%= render "actions_mobile" %>
-    </header>
+    </section>
```

Comment:
I think we need to make a few further tweaks to get the landmarks all in the place we want -

- This section is part of the main content of the page, so let's move it inside the `main`
- Excellent plan labelling the section, but I don't think this is necessary in all cases - when a section is labelled, it's added to the landmark regions for screen reader users. In this case I don't think the section is sufficiently important/standalone to merit that, so we can just remove the `aria-labelledby`

The same can be applied to the other areas below too

Reply:
Thanks for the detailed explanation. Learnt a few things here :) Honestly, I don't know much about accessibility, and I was just resolving the errors shown by axe. I have made the changes as requested. Kindly let me know if those are okay

Reply:
Thanks for making those changes! It's great to see those errors disappearing, and this makes things much better for landmark navigation for screen reader users 😄 

If you want to check out some examples of how screen readers allow for landmark navigation, there's [a nice article summarising this on w3](https://www.w3.org/TR/wai-aria-practices/examples/landmarks/at.html) - with your changes, people will now be able to get to the main content more easily, and not be confused by the duplicate header! 🌟